dev-python/virtualenvwrapper/virtualenvwrapper-4.8.4-r3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Set of extensions to Ian Bicking's virtualenv tool"
+HOMEPAGE="https://github.com/python-virtualenvwrapper/virtualenvwrapper
+	https://pypi.org/project/virtualenvwrapper/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/virtualenv[${PYTHON_USEDEP}]
+		dev-python/stevedore[${PYTHON_USEDEP}]
+		dev-python/virtualenv-clone[${PYTHON_USEDEP}]
+	')"
+BDEPEND="
+	${DEPEND}
+	$(python_gen_cond_dep '
+		dev-python/pbr[${PYTHON_USEDEP}]
+	')"
+
+PATCHES=(
+	"${FILESDIR}/${P}-adjust-test-error-message.patch"
+	"${FILESDIR}/${P}-dont-expect-easy_install.patch"
+	"${FILESDIR}/${P}-fix-wipeenv.patch"
+	"${FILESDIR}/${P}-remove-nositepackages-feature.patch"
+	"${FILESDIR}/${P}-remove-relocatable-tests.patch"
+)
+
+src_prepare() {
+	default
+	sed -i -e 's/egrep/grep -E/' "${S}/virtualenvwrapper.sh" || die
+
+	# specify default python interpeter to align with PYTHON_SINGLE_TARGET
+	sed -i -e \
+		"s|VIRTUALENVWRAPPER_PYTHON=\"\$(.*)\"|VIRTUALENVWRAPPER_PYTHON=\"${PYTHON}\"|" \
+		"${S}/virtualenvwrapper.sh" || die
+
+	# tox issues dont affect us + tries to FreeBSD(?) specific options for id (-P!)
+	sed -i -e '/# tox no longer exports/,$ d' "${S}/tests/setup.sh" || die
+
+	# remove tests which require an internet connection
+	rm tests/test_mkvirtualenv_install.sh || die
+	rm tests/test_mkvirtualenv_requirements.sh || die
+
+	# comment out tests which require functional git repos with remotes
+	sed -i -e '/test_wipeenv_pip_e () {/,/}/ s/^/#/' "${S}/tests/test_wipeenv.sh" || die
+	sed -i -e '/test_wipeenv_develop () {/,/}/ s/^/#/' "${S}/tests/test_wipeenv.sh" || die
+}
+
+python_compile() {
+	distutils-r1_python_compile
+	find "${BUILD_DIR}" -name '*.pth' -delete || die
+}
+
+python_test() {
+	# tests have unusual expectations due to setup.sh parsing id -P for HOME
+	export HOME="${HOME%/}"
+	export USER="${USER}"
+
+	cp -a "${BUILD_DIR}"/{install/usr,test} || die
+	export VIRTUAL_ENV="${BUILD_DIR}/test"
+
+	bash ./tests/run_tests "${VIRTUAL_ENV}" || die "Tests failed under ${EPYTHON}"
+}
+
+python_install_all() {
+	distutils-r1_python_install_all
+	find "${D}" -name '*.pth' -delete || die
```

Comment:
You're doing it twice now, also the eclass does it automatically these days.

Reply:
Noted, left them in from the previous revision as I was focused on the tests.